TCMManagement/DTOs/PatientCreation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;

```

Comment:
Add map from PatientCreation to Patient. When I did manual test, I found looks like it was not mapped properly. Did I miss something ?